hw2/05-form.html

```diff
@@ -7,14 +7,133 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
+  <body class="bg-dark">
+    <main class="container bg-white mt-5 py-4 rounded w-50">
+      <form id="userForm">
```

Comment:
I had difficulties with ignoring the html forms auto validation so I added novalidate to the form tag in my homework, which let me submit forms that don't list an email. I don't think that is necessary for this assignment, but because of the modal the error message doesn't appear when you don't enter an email.

Reply:
Thanks for catching that! Changes has been updated!